meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -314,6 +314,10 @@ RESIN_CONFIGS[xtables] = " \
     "
 
 # Deactivate the audit susbsystem and the audit syscall
+RESIN_CONFIGS_DEPS[audit] = " \
+    CONFIG_HAVE_AUDITSYSCALL=n \
+    CONIFG_SECURITY=n \
```

Comment:
This doesn't look like something we should do. There may be users of the BSP that rely on this feature